scripts/generator/graphql_generator/csharp/SDK/iOS/PaymentDeserializer.cs.erb

```diff
@@ -0,0 +1,52 @@
+#if UNITY_iPHONE
+namespace <%= namespace %>.SDK.iOS {
+    using System.Collections;
+    using System.Collections.Generic;
+    using <%= namespace %>;
+    using <%= namespace %>.MiniJSON;
+
+    public class PaymentDeserializer {
+
+        private enum PaymentField {
+            BillingContact,
+            ShippingContact,
+            ShippingIdentifier,
+            TokenData
+        }
+
+        private enum TokenField {
+            PaymentData,
+            TransactionIdentifier
+        }
+
+        public MailingAddressInput billingAddress { get; }
```

Comment:
Could we use `readonly` here instead of defining a `get` operator? Just curious.